docs/content/stable/yugabyte-platform/alerts-monitoring/anywhere-metrics.md

```diff
@@ -45,6 +45,23 @@ label_replace(
 
 For more information, refer to [Querying Prometheus](https://prometheus.io/docs/prometheus/latest/querying/basics/) in the Prometheus documentation.
 
+## Metrics collection level
+
+The metrics collection level setting controls how many database metrics are collected and stored. It helps balance system performance with monitoring needs, allowing administrators to optimize resource usage based on their specific observability requirements.
+
+The metrics collection level can be set via the **Metrics Collection Level** Universe Runtime Configuration option (config key `yb.metrics.collection_level`) to true. Refer to [Manage runtime configuration settings](../../../yugabyte-platform/administer-yugabyte-platform/manage-runtime-config/). You must be a Super Admin to set universe runtime configuration flags.
+
+The flag supports four distinct collection levels as described in the following table:
+
+| Level | Description |
+|:-------|:---------|
+| ALL | Collects all metrics from the database. |
+| NORMAL (default) | Collects all server-level metrics, and essential table-level metrics (required for YugabyteDB Anywhere UI and alerts). |
```

Comment:
I think we recently added another option called TABLE_OFF as part of https://yugabyte.atlassian.net/browse/PLAT-18210
And it's backported to 2025.1 and 2024.2 releases.
The option basically means we only scrape server level metrics.
Would be nice to add it right away if we're making a change, but only for above releases.